core/src/main/scala/org/typelevel/paiges/Doc.scala

```diff
@@ -318,8 +318,8 @@ sealed abstract class Doc extends Product with Serializable {
    * This method does not close `pw` or have any side-effects other
    * than the actual writing.
    */
-  def writeTo(width: Int, pw: PrintWriter): Unit = {
-    val it = Chunk.best(width, this)
+  def writeTo(width: Int, pw: PrintWriter, trim: Boolean = Doc.DEFAULT_TRIM): Unit = {
```

Comment:
Do you want me to do this here, or is this something you want in a different PR?